api/envoy/extensions/filters/network/thrift_proxy/v3/route.proto

```diff
@@ -30,6 +30,21 @@ message RouteConfiguration {
   // The list of routes that will be matched, in order, against incoming requests. The first route
   // that matches will be used.
   repeated Route routes = 2;
+
+  // An optional boolean that specifies whether the clusters that the route
+  // table refers to will be validated by the cluster manager. If set to true
+  // and a route refers to a non-existent cluster, the route table will not
+  // load. If set to false and a route refers to a non-existent cluster, the
+  // route table will load and the router filter will return a INTERNAL_ERROR
+  // if the route is selected at runtime. This setting defaults to true if the route table
+  // is statically defined via the :ref:`route_config
+  // <envoy_v3_api_field_extensions.filters.network.thrift_proxy.v3.ThriftProxy.route_config>`
+  // option. This setting default to false if the route table is loaded dynamically via the
+  // :ref:`trds
+  // <envoy_v3_api_field_extensions.filters.network.thrift_proxy.v3.ThriftProxy.trds>`
+  // option. Users may wish to override the default behavior in certain cases (for example when
+  // using TRDS with a static route table).
```

Comment:
@htuch 
What was wrong with the CDS in this comment?
In this form it is meaningless, TRDS means dynamic routing table.

Reply:
Sorry, you're right, let's revert.